BUILD.md

```diff
@@ -219,6 +219,10 @@ If you prefer to use an authenticated user, you can use a user or service accoun
 
 ## Miscellaneous
 
+### How to use devcontainers and a local Kubernetes cluster
+
+If you are working with devcontainers, Visual Studio Code is running from the container, so if you are using any local Kubernetes cluster like Kind or minikube, you don't have access from the container as default. To solve this and be able to work with devcontainers and a local cluster, you should follow [this official documentation from Microsoft](https://github.com/Microsoft/vscode-dev-containers/tree/main/containers/kubernetes-helm).
```

Comment:
The first phrase is a bit of an odd one and I'm not sure what you mean.

Is this what you mean?
```suggestion
If you are using Visual Studio Code to [develop inside a container (by using `devcontainers.json`)](https://code.visualstudio.com/docs/remote/containers) you can use any local Kubernetes cluster such as Kind or minikube. However, the development container will not have access by default.

To solve this and be able to work with `devcontainers.json`and a local cluster, you should follow [this official documentation from Microsoft](https://github.com/Microsoft/vscode-dev-containers/tree/main/containers/kubernetes-helm).
```

Reply:
I have rephrased the paragraph to be clearer, PTAL